Marlin/src/module/motion.cpp

```diff
@@ -562,7 +561,8 @@ void do_blocking_move_to(NUM_AXIS_ARGS(const float), const_feedRate_t fr_mm_s/*=
     const feedRate_t w_feedrate = fr_mm_s ?: homing_feedrate(W_AXIS)
   );
 
-  #if IS_KINEMATIC
+  #if IS_KINEMATIC && DISABLED(POLARGRAPH)
+    // kinematic machines are expected to home to a point 1.5x their range? never reachable.
```

Comment:
Is this referring to a Delta homing towards the top of the machine? If so, note that `position_is_reachable` only checks the x, y position to see if it's within the bed area, whether square or round-ish.